release-notes/9.0/preview/preview6/api-diff/Microsoft.NETCore.App/9.0-preview6_System.md

```diff
@@ -0,0 +1,32 @@
+# System
```

Comment:
Changes to System related to the System.Runtime area LGTM

CC. @dotnet/area-system-runtime